src/main/drivers/usb_msc.h

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
Can you please add the standard header to the files that are newly created in this pull request (see https://github.com/betaflight/betaflight/pull/5443/files#diff-513c9c337b40a964c5fda74016da3ff2L1)?

This does not apply to files copied from STM or other sources.